### PR TITLE
Use jupyter notebooks

### PR DESCRIPTION
**Pull request by ganochenkodg**
Vector databases basic guides: use Jupyter notebooks instead of running scripts in the python shell

### CODE REVIEW

**Review by bourgeoisor — COMMENTED** [review]
LGTM but let's not merge until we can also merge CL 622854406. Else, the doc page is going to be pointing to samples that don't exist anymore.

**Comment by ganochenkodg** [comment]
>LGTM but let's not merge until we can also merge CL 622854406. Else, the doc page is going to be pointing to samples that don't exist anymore.

Yep, it affects the current Qdrant guide, lets wait 👍

**Review by bourgeoisor — APPROVED** [review]
Snippet bot looks good now, LGTM!

**Pull request merged**
